account_analytic_project/model/account_analytic_account.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+# © 2015 Therp BV (http://therp.nl).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import _, api, fields, models
+
+
+class AccountAnalyticAccount(models.Model):
+    """Extend account.analytic.account."""
+    _inherit = 'account.analytic.account'
+
+    @api.multi
+    def create_project_for_account(self):
+        """Create a project for an already existing analytic account."""
+        project_model = self.env['project.project']
+        for this in self:
+            if not this.projects_ids and this.type != 'view':
```

Comment:
It is project_ids everywhere:
openeyedev@laptop:account-analytic$ cd account_analytic_project/
openeyedev@laptop:account_analytic_project$ grep -nr project_id * | grep -v Bina
model/account_analytic_account.py:26:        project_ids = self.project_ids
model/account_analytic_account.py:27:        assert project_ids, (_(
model/account_analytic_account.py:34:        domain = [('id', 'in', project_ids.ids)]
model/account_analytic_account.py:35:        res_id = project_ids[-1:].id
model/account_analytic_account.py:36:        if len(project_ids) > 1:
model/account_analytic_account.py:51:    project_ids = fields.One2many(
tests/test_account_analytic_project.py:11:            ('project_ids', '=', False),
tests/test_account_analytic_project.py:17:            account.project_ids)
view/account_analytic_account.xml:20:                            '|', ('project_ids', '!=', False),